servicetalk-examples/grpc/helloworld/build.gradle

```diff
@@ -33,7 +33,7 @@ serviceTalkGrpc {
 
   // The following setting must be omitted in users projects and is necessary here
   // only because we want to use the locally built version of the plugin
-  serviceTalkProtocPluginPath = "${project.rootProject.rootDir}/servicetalk-grpc-protoc/build/buildExecutable/" +
-      io.servicetalk.internal.build.ExecutableBuilder.addExecutablePostFix("protoc-gen-servicetalk_grpc",
-          org.gradle.internal.os.OperatingSystem.current().isWindows())
+  serviceTalkProtocPluginPath =
+      "${project.rootProject.rootDir}/servicetalk-grpc-protoc/build/buildExecutable/servicetalk-grpc-protoc-" +
+          project.version + "-all.jar"
```

Comment:
nit (here and in other places): You can avoid concatenation in this way: `"${project.rootProject.rootDir}/servicetalk-grpc-protoc/build/buildExecutable/servicetalk-grpc-protoc-${project.version}-all.jar"`

Reply:
I was getting a check style error in some cases bcz the line was too long so I'll just break it up for now.

Reply:
ok, np